cache/src/commonMain/kotlin/org/mobilenativefoundation/store/cache5/Cache.kt

```diff
@@ -21,6 +21,11 @@ interface Cache<Key : Any, Value : Any> {
      */
     fun getAllPresent(keys: List<*>): Map<Key, Value>
 
+    /**
+     * @return Map of the [Value] associated with each [Key] in the cache.
+     */
+    fun getAllPresent(): Map<Key, Value> = throw NotImplementedError()
```

Comment:
Added this for source compatibility of potential implementers of `Cache`. Let me know if you think this is unnecessary.

Reply:
I'm not sure if this is necessary, though appreciate you thinking about this. @digitalbuddha @yigit - thoughts?

Reply:
couldn't hurt